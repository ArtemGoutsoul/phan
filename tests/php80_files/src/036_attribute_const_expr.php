<?php
#[Attribute(Attribute::TARGET_FUNCTION | intdiv(2, 1))]
class X36 {
}

// PHP 8.0 attributes do not support named arguments
#[Attribute(flags: Attribute::TARGET_FUNCTION)]
class Y36 {
}
