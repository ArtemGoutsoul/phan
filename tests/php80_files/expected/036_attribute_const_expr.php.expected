%s:2 PhanInvalidConstantExpression Constant expression contains invalid operations
%s:7 PhanCompatibleNamedArgument Cannot use named arguments before php 8.0 in argument (flags: Attribute::TARGET_FUNCTION)
%s:7 PhanInvalidConstantExpression Constant expression contains invalid operations
